### PR TITLE
Link best practice doc part to secret management

### PR DESCRIPTION
**Pull request by noniagriconomie**
| Q             | A
| ------------- | ---
| License       | MIT
| Doc issue/PR  | 

Small addition, I find it usefull here

### CODE REVIEW

File: symfony/flex/1.0/.env [hunk 1]
```diff
@@ -9,6 +9,7 @@
 # Real environment variables win over .env files.
 #
 # DO NOT DEFINE PRODUCTION SECRETS IN THIS FILE NOR IN ANY OTHER COMMITTED FILES.
+# https://symfony.com/doc/current/best_practices.html#use-secret-for-sensitive-information
```
Comment (Nyholm) [inline symfony/flex/1.0/.env:12]: What about linking here directly?

https://symfony.com/doc/current/configuration/secrets.html

Comment (noniagriconomie) [reply symfony/flex/1.0/.env:12]: hi @Nyholm 
I wanted to point to the same page as the like below
But we can define the link that is more relevant for you :)

Comment (noniagriconomie) [reply symfony/flex/1.0/.env:12]: friendly ping @Nyholm

Comment (94noni) [reply symfony/flex/1.0/.env:12]: I’ll do it and swap and finish this PR
Thx for feed-back

Comment (noniagriconomie) [reply symfony/flex/1.0/.env:12]: done in https://github.com/symfony/recipes/pull/1074
closing here

**Pull request closed without merging**